pkg/generic/option.go

```diff
@@ -16,7 +16,11 @@
 
 package generic
 
-import "github.com/cloudwego/dynamicgo/conv"
+import (
+	"github.com/cloudwego/dynamicgo/conv"
+
+	"github.com/cloudwego/kitex/pkg/generic/thrift"
+)
 
 var (
 	DefaultHTTPDynamicGoConvOpts = conv.Options{
```

Comment:
should we also add MergeBaseFunc to http generic?

Reply:
yes